### ISSUE TITLE
Any plans to add tooltip on mouse over or keyboard shortcut to display help under the caret?

### ISSUE THREAD

**Issue by 7sharp9** [iraasta/autocomplete-elixir#16]
This would be really nice.

e.g:
![screen shot 2015-06-26 at 18 02 48](https://cloud.githubusercontent.com/assets/588746/8382894/130db184-1c2e-11e5-91ea-eecaae693243.png)

**Comment by iraasta** [comment]
Thought about it. The problem is where to fit it in

**Comment by 7sharp9** [comment]
You would have to parse the ident under the caret/mouse back to the module and function and then use Code.get_doc(Enum, :docs) then filter that to the function name atom.
A quick hack up would be something like:
```
Code.get_doc(Enum, :docs)
|>Enum.filter_map(&match?({{:filter_map, _}, _, _, _, _}, &1), fn ({_,_,_,_,doc}) -> doc end)
```

That excludes arity in the filter I think though.

**Comment by 7sharp9** [comment]
You could pin the current doc help as a notify popup like the scope cursor `ALT META P`
![screen shot 2015-06-26 at 18 25 27](https://cloud.githubusercontent.com/assets/588746/8383265/becce4c0-1c30-11e5-83f2-9e5e51e143c7.png)

**Comment by iraasta** [comment]
I know how to do that, but I'm more concerned about the usability. I think users won't like it in a popup. At least I wouldn't.

By the way feel free to modify and make pull requests. I'm open to collaboration :D

**Comment by 7sharp9** [comment]
Ok cool, see what I can do, need to look at what you have and get familiar first :-)

**Comment by iraasta** [comment]
Autocomplete-plus has something called `description` field in suggestion. It could fit in there I guess, but I can't get it to work. It just won't display
https://github.com/atom/autocomplete-plus/wiki/Provider-API

**Comment by 7sharp9** [comment]
Hmm, I see what you mean descriptionMoreURL is also available, I wonder why its not displayed.

**Comment by iraasta** [comment]
Sorry. Closed the wrong issue

**Comment by 7sharp9** [comment]
@iraasta Maybe the description field failing to display is due to the RSenseProviders recent depreciations.  I have another piece of code using autocompleteplus which returns a promise, If I set the description there then it displays.  Incidentally thats whats used by the Haskell autocomplete plus provider and the typescript provider too,

**Comment by 7sharp9** [comment]
Yep it looks like description is shown when the deprecated properties are adjusted.

Currently spec is empty so nothing is displayed at the bottom, is you switch the ordering so ret is first it is also displayed:

![screen shot 2015-06-28 at 18 28 39](https://cloud.githubusercontent.com/assets/588746/8397450/89ded6da-1dc3-11e5-82fc-62ba8c5911f8.png)

**Comment by iraasta** [comment]
I see. So how should I make it work? 
And do You think there's enough place for docs in there? 
I tried the notifications. This blinking drived me crazy ;)

**Comment by 7sharp9** [comment]
Best way to see is to hack it together I suppose, how could I easily add the doctext to `autocomplete.exs`?

Im still getting to grips with the code there.
